src/main/java/com/orasi/bluesource/EmployeePage.java

```diff
@@ -17,9 +15,17 @@
 	/**Page Elements**/
 	@FindBy(xpath = "//tr[1]//a[@class='glyphicon glyphicon-pencil']") Button btnEditFirstProject;
 	@FindBy(xpath = "//div[@id='panel_body_1']//table") Webtable tblProjectInfo;
-	@FindBy(xpath = "//button[@data-target='#modal_1']") Button btnEditGeneral;
-	@FindBy(xpath = "//div//a[contains(text(),'Deactivate Employee')]") Button btnDeactivateEmployee;
-	@FindBy(xpath = "//div[@class='panel-heading']//a[contains(text(),'Deactivate')]") Button btnDeactivate;
+	@FindBy(xpath = "//*[@id=\'accordion\']/div/div[7]/button") Button btnEditGeneral;
+	@FindBy(partialLinkText = "Deactivate Employee") Button btnDeactivateEmployee;
+	@FindBy(partialLinkText = "Deactivate") Button btnDeactivate;
+	@FindBy(xpath = "//*[@id=\"modal_1\"]/div/div") Label lblModal;
+	@FindBy(xpath = "//*[@id=\'employee_require_nonbillable\']") Button btnTimeEntryCheckbox;
+	@FindBy(id = "employee_status") Listbox lstStatus;
+	@FindBy(xpath = "//form/div[25]/input") Button btnUpdateEmployee;
```

Comment:
this xpath may be a bit flaky, since you are relying on it to be the 25th div on page, if they add more divs, then this will fail.  Something to think about.  Is there a unique parent or sibling element for this element, or something else that is more unique?